spec/section/annotations.md

```diff
@@ -62,7 +62,7 @@ Now, assume that we know:
 
 - Attribute `name` in the CSV is "UNIQUE" and "NOT NULL";
 
-Such a constraint naturally corresponds to the notion of `PRIMARY KEY` from the world of relational databases. This fact could be valuable information for the RML engine, especially in the virtual setting. However, note that constraints cannot be expressed on CSV files.
+Such a constraint naturally corresponds to the notion of `PRIMARY KEY` from the world of relational databases. This fact could be valuable information for the RML engine, especially in the virtual setting. However, note that constraints can not be expressed on CSV files.
```

Comment:
I believe "cannot" is more commonly used than "can not".

Reply:
Sure, but then it should be consistent, I'll adapt!